### ISSUE TITLE
Send multiple requests to single url in batch while testing

### ISSUE THREAD

**Issue by jatin33** [hoppscotch/hoppscotch#1583]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I have to send request to same url endpoint multiple times

**Describe the solution you'd like**
It would be great if we can build something where you can provide how many times a request should be triggered and then it automatically sends request at that mentioned url endpoint and show the api response of all requests being made.

**Describe alternatives you've considered**
Currently we have to manually send requests bu clicking **Send** button multiple times

**Additional context**
NA

**Comment by liyasthomas** [comment]
Thanks for the feature suggestion. You're referring to "chain multiple requests" and "show responses in batch" right? We'll consider this feature request ASAP.

**Comment by jatin33** [comment]
> Thanks for the feature suggestion. You're referring to "chain multiple requests" and "show responses in batch" right? We'll consider this feature request ASAP.

Yes exactly

**Comment by liyasthomas** [comment]
@jatin33 yes you can give it a try. Let me know if you need any help from our end. Assigning this issue to you.

**Comment by jatin33** [comment]
That sounds great, thanks @liyasthomas

**Comment by snapeur82** [comment]
Marci lov Bot Spike frech for me plaise

**Comment by snapeur82** [comment]
Love wour programme


Le jeu. 8 avr. 2021 13 h 09, Jatin Panjwani ***@***.***> a
écrit :

**Comment by jatin33** [comment]
@liyasthomas needed to discuss its design,

For starters, I think we can add a kebab menu on top of send button where we can provide options to configure request being sent
something like this
<img width="1064" alt="Screenshot 2021-04-10 at 10 03 05 PM" src="https://user-images.githubusercontent.com/13780467/114277380-8a16b980-9a48-11eb-83b2-6f1c12f834c6.png">

and when one of the options selected from kebab menu **Chain Multiple Requests** we should see input box below

<img width="1079" alt="Screenshot 2021-04-10 at 10 07 40 PM" src="https://user-images.githubusercontent.com/13780467/114277525-2ccf3800-9a49-11eb-8f9c-94a322beb1a9.png">

Let me know if you hame some different idea of implementing this

**Comment by liyasthomas** [comment]
@jatin33 this looks simple and very straightforward. I liked it. You can proceed with this design.

**Comment by jatin33** [comment]
great thanks, will try implementing this

**Comment by snapeur82** [comment]
Yessss

Le sam. 10 avr. 2021 13 h 41, Jatin Panjwani ***@***.***> a
écrit :

**Comment by EdwinFajardoBarrera** [comment]
Is this available? Is it has any progress? I would like to contribute.

**Comment by liyasthomas** [comment]
Hi @EdwinFajardoBarrera, seems like this feature request is not assigned to anyone - let me know how you'd approach this feature.

**Comment by EdwinFajardoBarrera** [comment]
Hi @liyasthomas, sorry for the waiting time, I have some ideas:

I would like to add the kebab menu on the right side of saving options (with a 'tooltip' with configuration meaning). I have the idea of adding inside this menu a number input (with default 1 as count) and adding each response in the result, maybe separated by a visual element. 

Another idea is to add another tab, especially for loop requesting. 

I would like to read your opinion and get some guidance about modules that could be implied in this implementation.

**Comment by calumk** [comment]
This would still be a good addition

**Comment by liyasthomas** [comment]
We're working on a "Collection Runner" feature that should achieve something very similar to this. Track here: #3600.

**Comment by gethari** [comment]
Running on loop for a single request is not available yet ?
